### ISSUE TITLE
Mock with useCamelCaseFlagKeys = false

### ISSUE THREAD

**Issue by tboulis** [launchdarkly/jest-launchdarkly-mock#6]
I've noticed in your code that you automatically convert the flags to kebab case and camelcase:
```
exports.mockFlags = function (flags) {
    mockUseFlags.mockImplementation(function () {
        var result = {};
        Object.keys(flags).forEach(function (k) {
            var kebab = lodash_kebabcase_1.default(k);
            var camel = lodash_camelcase_1.default(k);
            result[kebab] = flags[k];
            result[camel] = flags[k];
        });
        return result;
    });
};
```
We use flags with snake case. Is it possible to either configure this by passing a config object to `mockFlags` or by mocking the Provider that is returned from `asyncWithLDProvider`?

**Comment by tboulis** [comment]
I have created a PR locally that handles this case, but I cannot push it to this repo

**Comment by yusinto** [comment]
Hi @tboulis we have an upcoming release to support snake case so it's coming. I'll update this issue when the release is published. Thank you for your patience.

**Comment by yusinto** [comment]
This is now fixed in [v1.0.0](https://github.com/launchdarkly/jest-launchdarkly-mock/releases/tag/1.0.0).
